net-misc/turbovnc/turbovnc-2.2.5-r2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Use 'repoman commit' to fix these headers. If you copied it from somewhere, we need to know though, so we can figure out who has the rights to it.